server/odc-service/src/main/java/com/oceanbase/odc/service/projectfiles/ProjectFilesServiceFacade.java

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright (c) 2023 OceanBase.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.oceanbase.odc.service.projectfiles;
+
+import java.util.List;
+import java.util.Set;
+
+import org.springframework.stereotype.Service;
+
+import com.oceanbase.odc.service.projectfiles.model.BatchUploadProjectFileReq;
+import com.oceanbase.odc.service.projectfiles.model.FileUploadTempCredentialResp;
+import com.oceanbase.odc.service.projectfiles.model.GenerateProjectFileTempCredentialReq;
+import com.oceanbase.odc.service.projectfiles.model.ProjectFileMetaResp;
+import com.oceanbase.odc.service.projectfiles.model.ProjectFileResp;
+import com.oceanbase.odc.service.projectfiles.model.UpdateProjectFileReq;
+
+/**
+ * 项目文件管理服务类
+ *
+ * @author keyangs
+ * @date 2024/7/31
+ * @since 4.3.2
+ */
+@Service
+public class ProjectFilesServiceFacade implements IProjectFilesService {
```

Comment:
Facade should be an interface, we may define IProjectFilesService as facade, change current class to Impl class.

Reply:
Okay，I'll make this change.